trafilatura/htmlprocessing.py

```diff
@@ -226,6 +226,8 @@ def handle_textnode(
     preserve_spaces: bool = False,
 ) -> Optional[_Element]:
     "Convert, format, and probe potential text elements."
+    if is_image_element(elem):
```

Comment:
This will slow down processing, is it necessary?

Suggestion:
`if elem.tag == "graphic" and is_image_element(elem):`

Reply:
ok, fixed